mapzen-places-api/src/main/java/com/mapzen/places/api/ui/PlaceAutocompleteView.java

```diff
@@ -0,0 +1,99 @@
+package com.mapzen.places.api.ui;
+
+import com.mapzen.places.api.R;
+import com.mapzen.places.api.internal.PlaceAutocompleteActivity;
+
+import android.app.Activity;
+import android.app.Fragment;
+import android.content.Context;
+import android.content.Intent;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+/**
+ * Compound view which displays search icon, autocomplete text view and clear button.
+ */
+public class PlaceAutocompleteView extends LinearLayout {
```

Comment:
This class should not be part of the public API. Can it be moved to internal package or made package private scope?

Or as an alternative do we want to consider moving this class into `pelias-android-sdk` if we are going to be reworking the search view component that resides there? Could have one implementation that works in both places?